GRFC-TEMPLATE.md

```diff
@@ -2,7 +2,6 @@ Title
 ----
 * Author(s): [Author Name, Co-Author Name ...]
 * Approver: a11r
-* Status: {Draft, In Review, Ready for Implementation, Implemented}
 * Implemented in: <language, ...>
 * Last updated: [Date]
```

Comment:
Nit: Can we change "[Date]" to "YYYY-MM-DD", to make it clear that we expect ISO format here?

Reply:
Done